src/main/java/de/tum/in/www1/artemis/service/connectors/localci/buildagent/LocalCIDockerService.java

```diff
@@ -66,6 +67,10 @@ public class LocalCIDockerService {
     @Value("${artemis.continuous-integration.container-cleanup.cleanup-schedule-minutes:60}")
     private int containerCleanupScheduleMinutes;
 
+    // The image architecture that is supported by the build agent
+    @Value("${artemis.continuous-integration.image-architecture:amd64}")
```

Comment:
I would add that amd64 is the default architecture for most unix based systems because they are based on Intel CPUs